packages/ui/src/kit/VInputGroup/VInputGroup.vue

```diff
@@ -1,24 +1,47 @@
 <template>
-  <div :class="vInputGroupCssClass">
-    <slot />
+  <div>
+    <div :class="vInputGroupCssClass">
+      <slot />
+    </div>
+    <error-atom
```

Comment:
input-group is layout component, and no need to know about errors or anything else.
@kazurus what do you think?